src/Engine/Spells/Spells.h

```diff
@@ -140,6 +141,54 @@ enum SPELL_TYPE : uint8_t {
     SPELL_DISEASE = 153
 };
 
+/**
+ * Is spell target is item in inventory?
+ */
+inline bool IsSpellTargetsItem(SPELL_TYPE uSpellID) {
+    return uSpellID == SPELL_WATER_ENCHANT_ITEM ||
+           uSpellID == SPELL_FIRE_FIRE_AURA ||
+           uSpellID == SPELL_DARK_VAMPIRIC_WEAPON ||
+           uSpellID == SPELL_WATER_RECHARGE_ITEM;
+}
+
+/**
+ * Get skill used for casting given spell.
+ */
+inline PLAYER_SKILL_TYPE GetSkillTypeForSpell(SPELL_TYPE uSpellID) {
+    PLAYER_SKILL_TYPE result_skill;
+
+    assert(uSpellID != SPELL_NONE);
+
+    if (uSpellID < SPELL_AIR_WIZARD_EYE) {
+        result_skill = PLAYER_SKILL_FIRE;
```

Comment:
I suggest you drop the `result_skill` variable altogether and just use `return VALUE` inside the if blocks.